src/NumericalCalculations.jl

```diff
@@ -43,7 +43,7 @@ function _solveLambdaNumeric( fcttype::Union{F,<:Mixture{N_,F,S,T}},
   #
 
   #
-  r = nlsolve(objResX, u0, inplace=true)
+  r = nlsolve( (res, x) -> res .= objResX(x), u0, inplace=true)
```

Comment:
Ah, you are doing the opposite of what I was doing.  I had `residual` explicitly available for in-place operations (which we will still keep somewhere in `CalcFactor.residual::T` for hard in-place factor situations).

I see here now that you are changing to a more implicit usage of `residual` in the Roots (NLsolve.jl) case.  I did not see this before, sorry if I missed it.